### PR TITLE
Mvs 546 call to healthcheck in pat reg

### PR DESCRIPTION
**Pull request by adelk719**
## :floppy_disk: [Code Check]
_Make sure you've checked/done the following before making this PR!_
* Does your code build/run?
* Does your design approach make sense?
* Does your work achieve the intention of the card?
* Does your code have any unintentional side effects -- have you tested a full workflow & checked the logic branches you touched?
* Did you make a unit test for your code (as appropriate)?
* Is your branch relatively up to date with the Development branch (within ~2 commits)
* Have you moved the card to "Needs QA" in Jira

## :flipper: [JIRA Sprint Card]

https://mass-immunization-system.atlassian.net/browse/MVS-546

## :newspaper: [Work Description]

Added healthcheck functionality to patient registration app is that Success! is returned when patient info is submitted.

## :vertical_traffic_light: [Testing/QA Notes]

Follow the setup directions for the PatReg app in the readme.
Check if clicking "Yes," when submitting should return "Success!" in the web console

### CODE REVIEW

File: PatientRegistrationApp/src/App.vue [hunk 1]
```diff
@@ -90,7 +90,7 @@
 									<v-toolbar flat>
 										<v-toolbar-title class="text-wrap">Please ensure your information is correct</v-toolbar-title>
 									</v-toolbar>
-									<v-card flat><SinglePatientReviewSubmit/></v-card>
+									<v-card flat><SinglePatientReviewSubmit ref="singlepatientreviewsubmit"/></v-card>
```
Comment (axn9t9) [inline PatientRegistrationApp/src/App.vue:93]: May you follow camelCase for the ref here and for HouseholdReviewSubmit (and update the references)?

**Review by axn9t9 — CHANGES_REQUESTED** [review]

**Review by axn9t9 — COMMENTED** [review]
Looks good. Just a few nitpicks. Good work!

**Comment by rrucker3** [comment]
That sure is a lot of stuff to commit. Could you rebase on the current Development branch and remove the generated code? https://github.com/david-fraley/MassVaccinationSystem/pull/167 made it so that the generated code isn't pulled into the PR.

image

**Comment by axn9t9** [comment]
> That sure is a lot of stuff to commit. Could you rebase on the current Development branch and remove the generated code? #167 made it so that the generated code isn't pulled into the PR.
> 
> image

I would make a commit deleting the directories first, then rebase. If you rebase first, deleting the directories wont be tracked. Ran into this issue earlier.

**Review by axn9t9 — APPROVED** [review]

**Pull request merged**